neurobooth_os/tasks/s2s/task.py

```diff
@@ -0,0 +1,89 @@
+from __future__ import absolute_import, division
+import neurobooth_os
+import os.path as op
+from neurobooth_os.tasks.utils import make_win
+from neurobooth_os.tasks import utils
+from psychopy import visual
+from psychopy import prefs
+prefs.hardware['audioLib'] = ['pyo']
+
+
+class Sit_to_Stand():
```

Comment:
```suggestion
class SitToStand():
```

class names are camel case